netebpfext/user/fwp_um.h

```diff
@@ -25,6 +25,7 @@ typedef struct _fwp_classify_parameters
     uint64_t interface_luid;
     TOKEN_ACCESS_INFORMATION token_access_information;
     FWP_BYTE_BLOB user_id;
+    uint32_t reauth_flag;
```

Comment:
Avoid abbreviations.  Is it reauthentication_flag or reauthorization_flag?

Reply:
It is reauthorization_flag. Updated the name.